internal/jotlp/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package jotlp
```

Comment:
on reflection, I think we should name this package jpdata, since that's the name of the data model we use

Reply:
@yurishkuro since we're usually operating on `ptrace.Traces` - should we call it `jptrace`?

Reply:
ah, yeah, that's better